### DISCUSSION TITLE
Differentiate between lazer and stable scores?

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#23226] [Q&A] [answered]
Hello, I've been crudely keeping a record of my scores using the web API (to include failed/quit plays). However, i don't believe it accounts for things such as lazer DT having custom values (which to me, makes it strange that they get submitted to the web at all).

Ideally, I would like to make personal client that can query both lazer and stable databases and visualize score historical data (something like osu!alternative).

Anyway, I'm wondering if lazer score data is documented somewhere or how I can simply keep my data accurate.

**Comment by peppy** [comment, marked as answer]
if not already exposed, new mod customisation a will be added to the API in the coming future.
